packages/hw-app-btc/src/createTransaction.ts

```diff
@@ -77,14 +74,11 @@ export async function createTransaction(
     initialTimestamp,
     additionals,
     expiryHeight,
-    useTrustedInputForSegwit,
     onDeviceStreaming,
     onDeviceSignatureGranted,
     onDeviceSignatureRequested,
-  } = {
-    ...defaultsSignTransaction,
-    ...arg,
-  };
+  } = { ...defaultsSignTransaction, ...arg };
+  let { useTrustedInputForSegwit } = { ...defaultsSignTransaction, ...arg };
```

Comment:
that part is weird. we should put the first object in a variable to not spread it twice.